plugins/pulp_rpm/plugins/migrations/0020_nested_drpm_directory.py

```diff
@@ -0,0 +1,119 @@
+import glob
+import logging
+import os
+import shutil
+import tempfile
+
+import lxml.etree as le
+
+from pulp.server.config import config
+from pulp.server.db.connection import get_collection
+
+
+_logger = logging.getLogger('pulp_rpm.plugins.migrations.0020')
+
+
+def _move_nested_drpm_dir(repo_dir):
+    """
+    If a repository has a nested <repo_dir>/drpm/drpm/ directory,
+    remove one of the nestings as it should not have been created
```

Comment:
Can you add a bit more explaining why/when this might happen? It sounds strange that this would happen. Some context would be helpful to future readers.